internal/test/resources.go

```diff
@@ -1007,7 +1027,11 @@ func NewCoreEnvironmentVariables(minimal bool, tls bool, externalTLS bool, opens
 				Value: "custom-auth-cluster-role",
 			})
 		}
+		envs = append(envs, NewCoreAndGrafanaURLEnvironementVariables(minimal, tls, externalTLS)...)
+	} else if ingress { // On Kubernetes
+		envs = append(envs, NewCoreAndGrafanaURLEnvironementVariables(minimal, tls, externalTLS)...)
```

Comment:
Is this `ingress` branch useful? It seems like the same function call in both cases. Maybe I'm missing something though.

Reply:
Oh yes. I was adding this branch so that tests in k8s that do not have Ingress should not add external-url-related variable since the service won't be configured.

Reply:
Thanks, I understand now. I'm a bit torn about this. One one hand, Cryostat will not work without these environment variables. On the other, it's good to have one test that ensures that the operator doesn't crash if the Ingress configs are not specified on Kubernetes. I think this is fine for now, and we can revisit in #403.

Maybe calling the function `newNetworkEnvironmentVariables` might make it a bit clearer.

Reply:
Right! It would be better with that new name :D

Reply:
> think this is fine for now, and we can revisit in https://github.com/cryostatio/cryostat-operator/issues/403.

Oh yess! After that, I think I could remove this branch and these vars for both.